### PR TITLE
Implement coordinate transformation from geographic to Hycom convention

### PR DESCRIPTION
**Pull request by MariaAristizabal-NOAA**
The coordinate transformation is applied to the coast line points and the storm track. This is to ensure that the code that plots the different ocean fields works for all basins.

### CODE REVIEW

File: ush/python/ocean/MLDnc.py [hunk 1]
```diff
@@ -87,42 +90,80 @@
...
+
    fhr=k*6
    fig=plt.figure(figsize=(15,5))
-   plt.suptitle(storm.upper()+tcid.upper()+'  '+'Ver Hr '+"%3d"%(fhr)+'  (IC='+cycle+'): \n MLD & Change [m]',fontsize=15)
+   plt.suptitle(storm.upper()+tcid.upper()+'  '+'Ver Hr '+"%3d"%(fhr)+'  (IC='+cycle+'): '+var_name + ' & Change '+units,fontsize=15)
+
    ax121=plt.subplot(121)
-   var[k].plot.contourf(levels=np.arange(0,85,5),vmin=0,vmax=90,cmap='RdBu_r')
-   plt.plot(cx,cy,color='gray')
+   kw = dict(levels=np.arange(np.floor(np.nanmin(var)),np.ceil(np.nanmax(var)),delta_var))
```
Comment (JohnSteffen-NOAA) [inline ush/python/ocean/MLDnc.py:115]: Do we want to implement a "moving" colorbar scale that will change from storm to storm (as is coded here)? Or stick with pre-defined values in the original code?

File: ush/python/ocean/storm_HeatFlux.py [hunk 2]
```diff
@@ -98,166 +98,229 @@ def ZoomIndex(var,aln,alt):
...
 del xnc
 
-lns,lts=np.meshgrid(var1['longitude'],var1['latitude'])
-dummy=np.ones(lns.shape)
+lns,lts = np.meshgrid(varr1['longitude'],varr1['latitude'])
+dummy = np.ones(lns.shape)
+
+var_name1 = 'Latent Heat Flux'
+var_name2 = 'Sensible Heat Flux'
+var_name0 = 'Turbulence Heat Flux'
```
Comment (JohnSteffen-NOAA) [inline ush/python/ocean/storm_HeatFlux.py]: Naming convention should be update to: "Enthalpy Flux", "Net Turbulent Heat Flux" or something similar.

**Review by JohnSteffen-NOAA — COMMENTED** [review]
Hi Maria,
These code changes to the ocean hafs_graphics are a nice improvement. I've added a couple small suggestions/comments that do not affect the overall functionality. 

1. Whether we should automated the colorbar axis using "floor" and "ceiling" as these will values will change between different storms and basins.
2. Name change for the SHF+LHF to "Enthalpy Flux" or "Net Turbulent Heat Flux"

File: ush/python/ocean/MLDnc.py (hunk 1 above)
Comment (MariaAristizabal-NOAA) [reply ush/python/ocean/MLDnc.py:115]: I gave some consideration to this point and decided to propose a adjustable color bar range based on the minimum and maximum values for the specific variable to be plotted. The reason for this is that the range of values for different variables can change considerably depending on the basin. However, having a fixed color range for the different fields does facilitate the comparison of these fields between different storms and basins. So I suggest to give it a try to the adjustable range and then reconsider what option is best.

File: ush/python/ocean/storm_HeatFlux.py (hunk 2 above)
Comment (MariaAristizabal-NOAA) [reply ush/python/ocean/storm_HeatFlux.py]: I agree with the naming convention. I will change to "Enthalpy Flux"

**Review by JohnSteffen-NOAA — APPROVED** [review]

**Comment by BinLiu-NOAA** [comment]
Thanks @MariaAristizabal-NOAA and @JohnSteffen-NOAA! I am merging this PR back to the develop branch. Thanks!

**Pull request merged**